man/manual.md

```diff
@@ -8,20 +8,28 @@ documentclass: report
 
 # Introduction
 
-Cppcheck is an analysis tool for C/C++ code. It provides unique code analysis to detect bugs and focuses on detecting undefined behaviour and dangerous coding constructs. The goal is to detect only real errors in the code (i.e. have very few false positives). Cppcheck is designed to be able to analyze your C/C++ code even if it has non-standard syntax (common in embedded projects).
+Cppcheck is an analysis tool for C/C++ code. It provides unique code analysis to detect bugs and focuses on detecting 
+undefined behaviour and dangerous coding constructs. The goal is to detect only real errors in the code, and generate 
+as few false positives as possible. Cppcheck is designed to analyze your C/C++ code even if it has non-standard syntax, 
+as is common in for example embedded projects.
 
 Supported code and platforms:
 
-- You can check non-standard code that contains various compiler extensions, inline assembly code, etc.
-- Cppcheck should be compilable by any C++ compiler that handles the latest C++ standard.
+- Cppcheck checks non-standard code that contains various compiler extensions, inline assembly code, etc.
+- Cppcheck should be compilable by any compiler that supports the latest C++ standard.
 - Cppcheck should work on any platform that has sufficient CPU and memory.
 
-Please understand that there are limits of Cppcheck. Cppcheck is rarely wrong about reported errors. But there are
-many bugs that it doesn't detect.
+Cppcheck is rarely wrong about reported errors, but there are many bugs that it doesn't detect (yet). Most logarithmic 
```

Comment:
I am not sure I like these rewrited sentences.. could you leave them as is and then we take a closer look later. From here and down to `# Getting started`

Reply:
The words "Please understand" is addressing the reader directly, which I want to refrain from. I suggest to change it now and take a closer look later.

Reply:
ok

Reply:
so I will just ignore these lines and we change them later..